Sources/LiveKit/Protocols/RoomDelegate.swift

```diff
@@ -139,6 +139,10 @@ public protocol RoomDelegateObjC: AnyObject {
     /// Same with ``ParticipantDelegate/participant(_:didUpdate:permission:)``.
     @objc optional
     func room(_ room: Room, participant: RemoteParticipant, didUpdate publication: RemoteTrackPublication, permission allowed: Bool)
+
+    /// ``Room``'e2ee state has been updated.
+    @objc(room:publication:didUpdateE2EEState:) optional
+    func room(_ room: Room, publication: TrackPublication, didUpdate e2eeState: E2EEState)
```

Comment:
I was thinking of slowly migrating to a more explicit delegate signature,
for example didUpdateXXXXX instead of so many `didUpdate` delegates now.

Reply:
fixed 👇